### ISSUE TITLE
Create `ignore` setting for livepreview auto-reloading

### ISSUE THREAD

**Issue by andreamah** [microsoft/vscode-livepreview#548]
When you have `"livePreview.autoRefreshPreview": "On Changes to Saved Files"`, any changes to files end up reloading because we have a file watcher on the workspace files. It would be nice to have an ignore file for things that shouldn't trigger a reload automatically.

**Comment by Bloodiko** [comment]
Leaving this here as additional ressources for anyone who might look into this. 

Line in question
https://github.com/microsoft/vscode-livepreview/blob/06206789af441a46cfd3fcf39648a4066b0ef5c8/src/updateListener.ts#L30

[createFileSystemWatcher](https://code.visualstudio.com/api/references/vscode-api#workspace.createFileSystemWatcher:~:text=)%3A%20string-,createFileSystemWatcher,-(globPattern%3A)
 allows to match [Glob Patterns](https://code.visualstudio.com/api/references/vscode-api#GlobPattern)

Using an Array in the livePreview settings would allow for good customization. 
Ignoring .files (including folders) as well as node_modules (which is even mentioned explicitly in the documentation) may be default ?

**Comment by andreamah** [comment]
Related: https://github.com/microsoft/vscode-livepreview/issues/204

**Comment by lamrongol** [comment]
Yes, #691 is actually duplicated but I think ignoring `.git` folder is especially important.
I read back and #653 and noticed that is not related `.git` folder, However, problem originated from `.git` folder has two features: it's difficult that users notice files in `.git` is changed(by external apps) and files in `.git` folder doesn't affect project directly.

So at least, `ignore` setting has to have default setting to ignore `.git` folder.

**Comment by John-Dean** [comment]
This would be a really useful feature, particuarly due to the `.git` folder.

Can see this has been on the backlog for over a year, surely would just be a quick json setting for a blacklist and then matching this against the updated file.

As is mentioned above, just changing https://github.com/microsoft/vscode-livepreview/blob/79e9df2ffed927625988dfeb6876f84b087f12dd/src/updateListener.ts#L30 to use a different pattern, or blacklisting after like is done with the directory check would be very simple surely?
